doc/manual/manual.tex

```diff
@@ -9474,6 +9474,211 @@ \subsubsection{Yamauchi \& Takei anelastic shear wave velocity-temperature conve
+2571.9157 &
+-169676.4060 & 
+-169681.1065 &
+-169671.6199 \\
+&1  & $3^3$ &
+2562.6878 &
+2561.9847 &
+2563.7170 &
+-169676.3142 &
+-169676.8060 &
+-169675.9148 \\
+&2  & $4^3$  &
+2562.6993 &
+2562.6215 &
+2562.7395 &
+-169676.3211 &
+-169676.3361 &
+-169676.2903 \\
+& a.v. && 2562.6993 &&& -169676.3210 & &\\
+\hline\hline
+100&-1 & $1^3$ &
+2484.0821 &
+2479.3619 &
+2499.2301 &
+-164477.2574 &
+-164520.0786 &
+-164472.4899 \\
+&0 & $2^3$ & 
+2482.9027 &
+2481.7033 &
+2484.0834 &
+-164391.6151 &
+-164392.2246 &
+-164390.9963 \\
+&+1 & $3^3$ &  
+2482.8799 &
+2482.7731 &
+2482.9959 &
+-164391.6031 &
+-164391.6623 &
+-164391.5473 \\
+&+2 & $4^3$ & 
+2482.8819 &
+2482.8759 &
+2482.8865 &
+-164391.6041 &
+-164391.6067 &
+-164391.6012\\
+& a.v. & & 2482.8818 & & & -164391.6041 & &\\
+\hline\hline
+500&-1 & $1^3$&  
+2177.3562 &
+2177.2415 &
+2179.8737 &
+-144163.9859 &
+-144178.2736 &
+-144162.2361 \\
+& 0 & $2^3$ & 
+2176.2392 &
+2176.2253 &
+2176.2404 &
+-144088.7939 &
+-144088.7971 &
+-144088.7538 \\
+& 0 & $3^3$ & 
+2176.2391 &
+2176.2391 &
+2176.2392 &
+-144088.7937 & 
+-144088.7938 &
+-144088.7936 \\
+& 0 & $4^3$ & 
+2176.2391 &
+2176.2391 &
+2176.2391 &
+-144088.7937 &
+-144088.7937 &
+-144088.7937 \\
+& a.v. && 2176.2391 &&& -144088.7937 & &\\
+\hline\hline
+1500&-1 & $1^3$ & 
+1498.7997 &
+1498.7569 &
+1499.0522 &
+-99236.0403 &
+-99238.4122 &
+-99235.3167 \\
+& 0 & $2^3$ &
+1498.0239 &
+1498.0236 &
+1498.0240 &
+-99184.1673 &
+-99184.1678 &
+-99184.1647 \\
+& 1 & $3^3$ &
+1498.0240 &
+1498.0240 & 
+1498.0240 &
+-99184.1672 &
+-99184.1672 &
+-99184.1672 \\
+& 2 & $4^3$ &
+1498.0240 &
+1498.0240 &
+1498.0240 &
+-99184.1672 &
+-99184.1672 &
+-99184.1672 \\
+& a.v. && 1498.0240 &&& -99184.1672 &&\\
+\hline\hline
+3000 &-1 & $1^3$ &   
+717.8387 &
+717.8315 & 
+717.8533 &
+-47528.1777 &
+-47528.3488 &
+-47528.0725 \\
+ & 0 & $2^3$ &   
+717.4641 &
+717.4640 &
+717.4641 &
+-47503.2981 &
+-47503.2982 &
+-47503.2980 \\
+ & 1 & $3^3$ &   
+717.4641 &
+717.4641 &
+717.4641 &
+-47503.2981 &
+-47503.2981 &
+-47503.2981 \\
+& 2 & $4^3$ & 
+717.4641 &
+717.4641 &
+717.4641 &
+-47503.2981 &
+-47503.2981 &
+-47503.2981 \\
+&{\small a.v.} && 717.4641 &&& -47503.2981 && \\
+\hline
+\end{tabular}\\
+\caption{\it Thin shell gravity benchmark: $1\si{mGal}=10^{-5}\si{\metre\per\square\second}$. $n_q$ is the number of GLQ points per cell. 'a.v.' stands for analytical value.}
+\label{tab:thin_shell_gravity_benchmark}
+\end{table}
+
+The accuracy of the calculations increases with $I$ but so does the time spent in the postprocessor: for $I\in\{-1,0,1,2\}$  this time was about 18, 132, 440 and 1040~\si{s} respectively. This is easily explained when one realises that increasing $I$ from 0 to 1 means that the number of GLQ points per cell goes from $2^3=8$ to $3^3=27$, i.e. a $3.375$ increase in operations for the same number of cells and measurement points. Coincidentally, the time spent in the postprocessor increases by the same amount $440/132\sim 3.4$.
```

Comment:
```suggestion
The accuracy of the calculations increases with $I$ but so does the time spent in the postprocessor: for $I\in\{-1,0,1,2\}$  this time was about 18, 132, 440 and 1040~\si{s} respectively. This is easily explained when one realises that increasing $I$ from 0 to 1 means that the number of GLQ points per cell goes from $2^3=8$ to $3^3=27$, i.e. a $3.375$ increase in operations for the same number of cells and measurement points. The time spent in the postprocessor increases by a similar factor ($440/132\sim 3.4$).
```

Reply:
You could also quantify the relative accuracies when using the different values of $I$ here.

Reply:
```suggestion
The accuracy of the calculations increases with $I$ but so does the time spent in the postprocessor: for $I\in\{-1,0,1,2\}$  this time was about 18, 132, 440 and 1040~\si{s} respectively. This is easily explained when one realizes that increasing $I$ from 0 to 1 means that the number of GLQ points per cell goes from $2^3=8$ to $3^3=27$, i.e. a $3.375$ increase in operations for the same number of cells and measurement points. Coincidentally, the time spent in the postprocessor increases by the same amount $440/132\sim 3.4$.
```

Reply:
this benchmark is part of a paper about to be submitted in which we discuss the errors and compare those with errors obtained with other codes/methods.